docs/_src/api/pydoc/transformers-img-to-text.yml

```diff
@@ -0,0 +1,26 @@
+loaders:
+  - type: python
+    search_path: [../../../../haystack/nodes/image_to_text]
+    modules: ['base', `transformers`]
```

Comment:
Hey @agnieszka-m it looks like the [Generate API docs](https://github.com/deepset-ai/haystack/actions/runs/4022654235/jobs/6912620782) CI is complaining about the "`" character used around the "transformers" word. I think it needs to be changed to 
```
'transformers'
```